C/CeresSolver/build_tarballs.jl

```diff
@@ -37,17 +37,12 @@ products = Product[
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    # CeresSolver is removing OpenMP and dependencies for OpenMP are dropped
-    # https://github.com/ceres-solver/ceres-solver/issues/886
-    # Eigen_jll v0.3.4 throws errors on powerpc64le with older GCC versions
     BuildDependency("Eigen_jll"),
     Dependency("glog_jll"),
-    # Metis replaces SuiteSparse on Windows
     Dependency("METIS_jll"),
-    Dependency("OpenBLAS32_jll"),
-    # Hard code the version now as the latest v7.0.1 does not get recognized
-    Dependency("SuiteSparse_jll", v"5.10.1")
+    Dependency("libblastrampoline_jll"),
+    Dependency("SuiteSparse_jll", v"7.2.1")
```

Comment:
Please use compat bounds, not just build version numbers, they're almost useless
```suggestion
    Dependency("libblastrampoline_jll"; compat="5.4.0"),
    Dependency("SuiteSparse_jll"; compat="7.2.1"),
```

Reply:
Ok - now I understand what you mean.